test-storybooks/portable-stories-kitchen-sink/react/.storybook/main.ts

```diff
@@ -37,6 +25,15 @@ const config: StorybookConfig = {
   viteFinal: (config) => {
     return {
       ...config,
+      optimizeDeps: {
+        ...config.optimizeDeps,
+        include: [
+          ...(config.optimizeDeps?.include || []),
+          "react-dom/test-utils",
+          "@storybook/react/**",
+          "@storybook/experimental-addon-test/preview",
+        ],
+      },
```

Comment:
potentially not needed anymore, just please run the react kitchen sink project and make sure it doesn't take a long time loading as it can cause flake on E2E